src/components/AvatarCropModal/AvatarCropModal.tsx

```diff
@@ -177,7 +172,7 @@ function AvatarCropModal(props) {
      * @param {Number} newY
      */
     const updateImageOffset = useWorkletCallback(
```

Comment:
```suggestion
    /**
     * Validates the offset to prevent overflow, and updates the image offset.
     */
    const updateImageOffset = useWorkletCallback(
```

Also, please remove `* @returns {Object}` from `getDisplayedImageSize`